Sources/SWBCore/SpecImplementations/Tools/SwiftCompiler.swift

```diff
@@ -3397,10 +3401,12 @@ public final class SwiftCompilerSpec : CompilerSpec, SpecIdentifierType, SwiftDi
             removeWithParameter(arg)
         }
 
-        // We need to ignore precompiled headers due to:
+        // For some old version of swift driver, the output path for bridging header pch is not stable,
```

Comment:
Do we have any compatibility requirement for older swift-driver? If we only need to support recent swift-driver, this workaround can be removed to make preview build faster

Reply:
The change in swift-driver I am talking about is: https://github.com/swiftlang/swift-driver/pull/1832

It has been few months but not in official release yet.